user/models/__init__.py

```
@@ -0,0 +1,15 @@
+from .base import (
+    User,
```

Ruff (0.12.2)

`.base.User` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)